### ISSUE TITLE
[ACCESSIBILITY] Explorations of Options to Improve Check-in Poster

### ISSUE THREAD

**Issue by kristenmcconnell** [department-of-veterans-affairs/va.gov-team#30440]
# Description
Starting the mobile checkin process with a poster can be challenging for low-vision Veterans. We need to explore ways to improve this experience.

Already gathered ideas:
- [Product found by Martha Wilkes](https://dsva.slack.com/archives/C022AC2STBM/p1630437213141700)
- De-prioritized previous ideas: pushing a text message to Veterans and geolocation.

# Tasks 
- [ ] Talk to @artsymartha68, who has been building accessibility relationships across VA and helping out on the check-in project
- [ ] Explore options
- [ ] Create recommendations

# Acceptance Criteria
- [ ] Share with team

**Comment by julieadair1** [comment]
Goal for this sprint - have initial conversations with Martha Wilkes.
Next sprint determine options.

**Comment by niradatta1** [comment]
@stephenBarrs Great to hear! So here is[ a google document ](https://docs.google.com/document/d/1i1sqKzU_t_L8V-1pdNcG8bxUnL9bNwu2GVSFEADNp-4/edit#)I started to capture all the conversations I'm having with the accessibility team, and a very early set of ideas on possible solutions. To @kristenmcconnell we definitely don't have a solution yet. However, we've identified a next step to map out all the accessibility use cases with each potential solution, so we can recommend a combination of alternatives to the poster to ensure we're meeting the majority of needs covered by a diversity of abilities. I will spin this work up as a separate ticket and tag accordingly. @ytsaoca and I plan to pick that up at an upcoming sprint.
